src/ng_package/text_replace/index.ts

```diff
@@ -0,0 +1,64 @@
+import fs from 'node:fs/promises';
```

Comment:
Nit: Can you use `.mts` or `.ts` for these files? That avoids weird errors with Node accidentally running as `CJS` or `ESM`. I think we should use `.mts` here, as I did elsewhere or in other rules I've recently built